app/frontend/src/store/index.js

```diff
@@ -1,10 +1,12 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 
+import form from '@/store/modules/form.js';
+
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  modules: {},
+  modules: { form },
```

Comment:
I'm still slightly wary about having this always be mounted - it's quite possible for a user to just visit the about page or myForms page and end up doing nothing else -> in this situation they'd never need the form module tree ever mounted. Honestly can go both-ways, but anytime we could leverage module tree-shaking I think we should be taking advantage of it.

Reply:
Discussed